src/components/btn-alt/btn-alt.scss

```
@@ -0,0 +1,111 @@
+$light-grey: #f4f4f4;
+$grey: #bcbcbc;
+$dark-grey: #909090;
+
+$white: #ffffff;
+
+$light-blue: #99b3d8;
+
+$light-red: #ff0000;
+$red: #bf1d26;
+$dark-red: #81080f;
+
+$desktop: 1280px;
+$tablet: 768px;
+$mobile: 375px;
+
+.btn-alt {
+	$root: &;
+	height: 48px;
+
+	box-sizing: border-box;
+
+	display: flex;
+	align-items: center;
+
+	padding: 0 14px 0 18px;
+
+	border-radius: 25px;
+	background-color: $light-red;
+	color: $white;
+
+	transition:
+		color 0.15s ease-in-out,
+		background-color 0.15s ease-in-out;
+
+	.btn-alt__text {
+		font-family: Arial, sans-serif;
+		font-size: 16px;
+		font-weight: 400;
+		letter-spacing: -0.32px;
+		margin-right: 14px;
+	}
+
+	svg {
+		width: 32px;
+		height: 32px;
+		fill: $white;
+
+		transition: fill 0.15s ease-in-out;
+	}
+
+	&:active {
+		background-color: $dark-red;
+	}
+
+	&:disabled {
+		color: $dark-grey;
+		background-color: $light-grey;
+		pointer-events: none;
+
+		svg {
+			fill: $dark-grey;
+		}
+	}
+
+	@media (max-width: $desktop) {
+		& {
+			height: 42px;
+
+			.btn-alt__text {
+				font-size: 14px;
+			}
+
+			svg {
+				width: 24px;
+				height: 24px;
+			}
+		}
+	}
```

В данном случае такая вложенность не обязательна
```suggestion
	@media (max-width: $desktop) {
	    height: 42px;
    
	    .btn-alt__text {
		font-size: 14px;
	    }
    
	    svg {
		width: 24px;
		height: 24px;
	    }

	}
```